defaults/config.js

```diff
@@ -86,6 +86,28 @@ module.exports = {
 	//
 	prefetchStorage: false,
 
+	//
+	// Allow uploading files to the server hosting TheLounge.
+	// If enabled, The Lounge will store files in the `${THELOUNGE_HOME}/storage` folder.
+	//
+	// Images will not expire or be removed by TheLounge, so be wary of memory and your hardware.
+	//
+	// @type     boolean
+	// @default  false
+	//
+	upload: false,
+
+	//
+	// Maximum file size for uploaded files
+	//
+	// If upload is enabled, The Lounge will only allow uploading files with a size < the value below
+	// Specified value is in kilobytes. Default value is 10240 bytes (10 MB).
```

Comment:
Default value is *kilo*bytes.

Also probably should be 10000 since it gets multiplied by 1024? (I know I said 10240 myself before).

Reply:
As other config values are specific (prefetch is 2048), i'll instead multiply by 1000 for consistency.

Reply:
Done (my comment)